### PR TITLE
Cleanup a few unused functions

### PR DESCRIPTION
**Pull request by pepin-stripe**
### Reviewers
r? @stripe/developer-products 
cc @stripe/developer-products

 ### Summary
Cleaning up a few unused functions before start work on the Cassette DX

### CODE REVIEW

File: pkg/playback/http_playback_test.go [hunk 1]
```diff
@@ -140,19 +138,18 @@ func TestSimpleRecordReplayServerSeparately(t *testing.T) {
...
-	resShutdown, err := http.Post("http://localhost:8080/playback/stop", "text/plain", nil)
+	resShutdown, err := http.Get("http://localhost:8080/playback/cassette/eject")
 	server.Shutdown(context.TODO())
 	assert.NoError(t, err)
 	defer resShutdown.Body.Close()
 
 	// --- Set up a replay server
-	httpReplayer := newReplayServer(webhookURL)
-	err = httpReplayer.readCassette(&cassetteBuffer)
+	httpWrapper, err = NewServer(remoteURL, webhookURL, "/tmp", Replay, "cassette.yaml")
```
Comment (suz-stripe) [inline pkg/playback/http_playback_test.go:147]: might want to change this dir name to use `os.TempDir` to ensure Windows tests pass

**Review by suz-stripe — CHANGES_REQUESTED** [review]

File: pkg/playback/http_playback_test.go (hunk 1 above)
Comment (pepin-stripe) [reply pkg/playback/http_playback_test.go:147]: oh good point!

**Comment by pepin-stripe** [comment]
ptal @suz-stripe

**Review by suz-stripe — APPROVED** [review]
LGTM 🎉  thanks for fixing the test 🙆🏼‍♀️

**Pull request merged**